test/post-911-gib-status/components/EnrollmentHistory.unit.spec.jsx

```diff
@@ -52,12 +53,14 @@ describe('<EnrollmentHistory>', () => {
     expect(featureBoxes[0].text()).to.equal('Does something look wrong in your enrollment history?');
   });
 
-  it('should show history may be incorrect warning if benefit pending', () => {
+  it('should show history may be incorrect warning if benefit used up', () => {
```

Comment:
Changed this to be checking for whether they are active duty, which is what the front-end hinges on now.